roles/acm_setup/tasks/main.yml

```diff
@@ -181,13 +181,16 @@
 
 - name: "Update the provisioning resource to watch all Namespaces"
   community.kubernetes.k8s:
+    merge_type:
+      - merge
     definition:
       apiVersion: metal3.io/v1alpha1
       kind: Provisioning
       metadata:
         name: provisioning-configuration
       spec:
         watchAllNamespaces: true
+        disableVirtualMediaTLS: true
```

Comment:
@betoredhat, I'd like to know you opinion on whether this can be left like this or it should be made optional.

Reply:
@nsilla sure, let's add a variable to control this. I haven't seen many environments where the BMCs have verified TLS certificates.